contao/helper/exception.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+
+/**
+ * Class UnusedArgumentsException
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class UnusedArgumentsException extends Exception {}
+
+
+/**
+ * Class UnresolvableDependenciesException
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class UnresolvableDependenciesException extends Exception {}
```

Comment:
We are in fact still using it. It can only be marked as deprecated once we deprecate the Contao class loader.